plugins/rts54/rts54.quirk

```diff
@@ -0,0 +1,16 @@
+# RTS5423
+[DeviceInstanceId=USB\VID_0BDA&PID_1100]
+Plugin = rts54
+FirmwareSizeMin = 0x10000
+FirmwareSizeMax = 0x10000
+Children = FuRts54Module|USB\VID_0BDA&PID_1100&I2C_01
```

Comment:
Have you tested this child getting created properly?  In my plugin I tried to do something similar, but the type doesn't register unless I call `g_type_ensure`.

Reply:
> Have you tested this

Nope, I'm still waiting for hardware and specifications.

> unless I call g_type_ensure

Ahh, right. I guess that makes sense. I'm wondering if the daemon can do something clever here, rather than requiring plugins to manually `g_type_ensure()` subclasses.

> child never gets probe/setup called this way

I guess we can do that automatically. Will think.

Reply:
> I'm wondering if the daemon can do something clever 

Nope, we really have to do this manually -- we can't use `g_type_ensure()` until the type has actually been registered.

Reply:
OK thanks, that matches what I was seeing too.